app/policies/volunteer_policy.rb

```diff
@@ -10,4 +10,12 @@ def initialize(user, record)
   def update_volunteer_email?
     user.casa_admin?
   end
+
+  def activate?
+    @user.casa_admin? || @record.supervisor == @user
```

Comment:
Note: Zoe says any supervisor should be able to do this- I will change this shortly. :)

Reply:
done